core/contracts/common/implementation/Testable.sol

```diff
@@ -31,10 +31,10 @@ abstract contract Testable {
     /**
      * @notice Sets the current time.
      * @dev Will revert if not running in test mode.
-     * @param _time timestamp to set current Tesable time to.
+     * @param time timestamp to set current Tesable time to.
      */
-    function setCurrentTime(uint _time) external onlyIfTest {
-        Timer(timerAddress).setCurrentTime(_time);
+    function setCurrentTime(uint time) external onlyIfTest {
+        Timer(timerAddress).setCurrentTime(time);
```

Comment:
Why were these changed to *not* have preceding underscores but the previous file was changed to add preceding underscores to function args?

Reply:
I think this is consistent with the rule "parameters never have underscores except in constructors."

Reply:
Ahh, I see. Ignore the above. I got my wires crossed. Thought this was a constructor.

Reply:
yup, that's right @nicholaspai